templates/events/show.html.twig

```diff
@@ -3,6 +3,10 @@
 {% block page_title event.name %}
 {% block canonical_url url('app_event_show', {'slug': event.slug}) %}
 
+{% block meta_description %}
+    {{ event.name }} est un événement local du mouvement La République En Marche ! organisé par {{ event.organizer ? event.organizer.partialName : '' }} le {{ event.beginAt|localizeddate('full', 'none') }} à {{ event.beginAt|date('G\\hi') -}}, dans la ville de {{ event.postAddress.cityName }}. Visitez la page dédiée pour vous inscrire et obtenir toutes les informations concernant cet événement.
```

Comment:
If event organizer does not exist it means that adherent was unregistered. It's better to display `un ancien adhérent` instead of empty string.